content/blog/pulumi-kubernetes-operator-2-0/index.md

```diff
@@ -0,0 +1,269 @@
+---
+date: "2024-10-17"
+title: "Pulumi Kubernetes Operator 2.0"
+authors: ["eron-wright"]
+tags: ["Kubernetes", "Continuous-Delivery", "operators"]
+meta_desc: "Pulumi Kubernetes Operator 2.0: Horizontal Scaling, Multi-Tenancy"
+meta_image: operator.png
+---
+
+A few years ago we released the [Pulumi Kubernetes Operator](/blog/pulumi-kubernetes-operator-1-0/), a cloud-native way to manage and deploy cloud infrastructure using Pulumi from within your Kubernetes environment. We've heard your feedback about limitations related to scalability and isolation.
```

Comment:
In the spirit of telling them what you're going to tell them, it may also make sense to mention Robustness and that the changes effect Operability and installation here.